python/orca/example/learn/tf2/yolov3/yoloV3.py

```diff
@@ -575,33 +577,36 @@ def main():
-                                "spark.kubernetes.driver.volumes.persistentVolumeClaim."
-                                "nfsvolumeclaim.mount.path": options.nfs_mount_path})
-    elif options.cluster_mode.startswith("yarn"):
-        init_orca_context(cluster_mode=options.cluster_mode, cores=options.cores,
-                          num_nodes=options.worker_num, memory=options.memory,
-                          init_ray_on_spark=True, enable_numa_binding=options.enable_numa_binding,
-                          object_store_memory=options.object_store_memory)
-    elif options.cluster_mode == "spark-submit":
-        init_orca_context(cluster_mode="spark-submit")
+    if options.runtime == "ray":
+        init_orca_context(runtime=options.runtime, address="localhost:6379")
+    else:
+        if options.cluster_mode == "local":
+            init_orca_context(cluster_mode="local", cores=options.cores, num_nodes=options.worker_num,
+                            memory=options.memory, init_ray_on_spark=True,
+                            object_store_memory=options.object_store_memory)
+        elif options.cluster_mode == "k8s":
+            init_orca_context(cluster_mode="k8s", master=options.k8s_master,
+                            container_image=options.container_image,
+                            init_ray_on_spark=True, enable_numa_binding=options.enable_numa_binding,
+                            num_nodes=options.worker_num, cores=options.cores, memory=options.memory,
+                            object_store_memory=options.object_store_memory,
+                            conf={"spark.driver.host": options.driver_host,
+                                    "spark.driver.port": options.driver_port,
+                                    "spark.kubernetes.executor.volumes.persistentVolumeClaim."
+                                    "nfsvolumeclaim.options.claimName": "nfsvolumeclaim",
+                                    "spark.kubernetes.executor.volumes.persistentVolumeClaim."
+                                    "nfsvolumeclaim.mount.path": options.nfs_mount_path,
+                                    "spark.kubernetes.driver.volumes.persistentVolumeClaim."
+                                    "nfsvolumeclaim.options.claimName": "nfsvolumeclaim",
+                                    "spark.kubernetes.driver.volumes.persistentVolumeClaim."
+                                    "nfsvolumeclaim.mount.path": options.nfs_mount_path})
+        elif options.cluster_mode.startswith("yarn"):
+            init_orca_context(cluster_mode=options.cluster_mode, cores=options.cores,
+                            num_nodes=options.worker_num, memory=options.memory,
+                            init_ray_on_spark=True, enable_numa_binding=options.enable_numa_binding,
+                            object_store_memory=options.object_store_memory)
+        elif options.cluster_mode == "spark-submit":
```

Comment:
I think the default behavior of `init_orca_context` should be the same as "spark-submit"?

Reply:
> I think the default behavior of `init_orca_context` should be the same as "spark-submit"?

Yes, when `activate spark context`.

Reply:
> I think the default behavior of `init_orca_context` should be the same as "spark-submit"?

Only when there is an activate spark context, the default behavior is the same as "spark-submit".

Reply:
> I think the default behavior of `init_orca_context` should be the same as "spark-submit"?

Yes, when there is an activate spark context.